### DISCUSSION TITLE
Shimmer Staking rewards werden nicht gefunden

### DISCUSSION THREAD

**Discussion by Mikele89** [iotaledger/firefly#7640] [Help & Support]
Staking Rewards werden nicht gefunden egal wie oft ich sie suche. Suche erfolgt über Recovery Phrase, Anmeldung funktioniert aber gefunden wird dann nichts, egal wie oft ich suche. Such nach dem Anlegen des Profils bei Firefly keine Möglichkeit mehr diese zu finden.

**Comment by begonaalvarezd** [comment]
We found a bug and are working on a new release to fix it, it will be out soon

**Comment by begonaalvarezd** [comment]
Hi, we released a new version fixing this issue  🙏 please upgrade to the new version. We would appreciate if you let if know if the issue persists or not 🌷 thanks!
